system/doc/reference_manual/expressions.xml

```diff
@@ -1348,6 +1348,27 @@ handle_call(change, From, #{ state := start } = S) ->
   <section>
     <marker id="bit_syntax"></marker>
     <title>Bit Syntax Expressions</title>
+    <p>The bit syntax operates on <em>bit strings</em>.</p>
+    <p>
+      A <em>bit string</em> is a sequence of bits stored in
+      a sequence of (8-bit) bytes in memory address order.
```

Comment:
A young reader may ask what other kind of bytes there are.

```suggestion
      a sequence of bytes (octets) in memory address order.
```

Reply:
Agreed